DTS-STN/secure-client-hub
# Update main from dev (September 22, 2023)

### Description of proposed changes:
This PR updates main to be in line with the latest changes from dev

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure app runs as expected
